src/components/atoms/LanguageOption/index.tsx

```diff
@@ -24,13 +24,16 @@ const setLangVer = (props) => {
 const LanguageOption: React.FC<LanguageOptionInterface> = (
     props: LanguageOptionInterface
 ) => {
-    console.log("HEEE: ", props.language)
     return (
         <StyleLanguageOptionContainer>
             {props.language === "en" || props.language === "English" ? 
                 <StyleLanguageLink to={`/ja${location.pathname}`} onClick={setLangVer}>{japanFlagSvg}</StyleLanguageLink>
              :
+             props.urlSlug === "/ja" 
+                ? 
                 <StyleLanguageLink to={`/${props.urlSlug}`} onClick={setLangVer}>{americanFlagSvg}</StyleLanguageLink>
+                :
+                <StyleLanguageLink to={`/`} onClick={setLangVer}>{americanFlagSvg}</StyleLanguageLink>
```

Comment:
What's the purpose of this line? 

I don't think that you need other conditions, just two is fine.

```
props.urlSlug === 'ja'
// do X
: 
// do Y (assuming that this is the case where the props are NOT 'ja' ('en', 'es', 'fr')
```

Reply:
Also, what if the language is in Spanish or in other languages besides English or Japanese?